src/Operation/Distinct.php

```diff
@@ -175,6 +186,11 @@ private function createCommandDocument(): array
             $cmd['collation'] = (object) $this->options['collation'];
         }
 
+        if (isset($this->options['hint'])) {
+            /** @psalm-var string|array|object */
```

Comment:
Do you really need this annotation? If it applies to `$cmd['hint']`, it should be `string|object`, without `array`

Reply:
Psalm did complain with out this. Good catch, though, as I can remove `array` here.